lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -251,55 +251,59 @@ def verify
     res
   end
 
-  def add(source, dest_uri)
+  def add(*upload_args)
```

Comment:
It looks like most of the content was removed. Can we just inherit the whole method or are we keeping it just for the logging?

Reply:
I figured someone added the logging for a specific purpose, and didn't want to blind side them by removing it.

But yes, I was just keeping it for the logging mostly...

...

Though, now that I look at it, I also handled the `reconnect!` and `disconnect` here, and it makes sense that it is done as part of `.add` and not trying to do it for each file in `.upload_single` (assuming the many "splits" case).